### ISSUE TITLE
No "MigrateHamcrestToAssertJ" recipe is executed or changed in the entire test class

### ISSUE THREAD

**Issue by Saturas89** [openrewrite/rewrite-testing-frameworks#521]
## What version of OpenRewrite are you using?
I am using

- Maven plugin v5.32.1
- rewrite-testing-frameworks v2.10.1

## How are you running OpenRewrite?
I am using the Maven plugin, and my project is a single module project.
```xml
<plugin>
  <groupId>org.openrewrite.maven</groupId>
  <artifactId>rewrite-maven-plugin</artifactId>
  <version>5.32.1</version>
  <configuration>
    <activeRecipes>                                                    
      <recipe>org.openrewrite.java.testing.hamcrest.MigrateHamcrestToAssertJ</recipe>
    </activeRecipes>
  </configuration>
</plugin>
```

## What is the smallest, simplest way to reproduce the problem?
We have a simple testclass like this:

```java
package *.model.internal;

import org.junit.Test;

import static org.hamcrest.core.IsEqual.equalTo;
import static org.hamcrest.core.IsNot.not;
import static org.hamcrest.MatcherAssert.assertThat;

public class IdentValueStringTest {

    private IdentValueString value = new IdentValueString();
    @Test
    public void testEquals() throws Exception {
        IdentValueString clone = value.clone();

        assertThat(value, equalTo(clone));

        value.setValue("anders");
        assertThat(value, not(equalTo(clone)));
    }
```

## What did you expect to see?

That the methods and imports are replaced.

```java
package *.model.internal;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class IdentValueStringTest {

    private IdentValueString value = new IdentValueString();
    @Test
    public void testEquals() throws Exception {
        IdentValueString clone = value.clone();

        assertThat(value).isEqualTo(clone);

        value.setValue("anders");
        assertThat(value).isNotEqualTo(clone);
    }
```

## What did you see instead?

Nothing changed.

```java
package *.model.internal;

import org.junit.Test;

import static org.hamcrest.core.IsEqual.equalTo;
import static org.hamcrest.core.IsNot.not;
import static org.hamcrest.MatcherAssert.assertThat;

public class IdentValueStringTest {

    private IdentValueString value = new IdentValueString();
    @Test
    public void testEquals() throws Exception {
        IdentValueString clone = value.clone();

        assertThat(value, equalTo(clone));

        value.setValue("anders");
        assertThat(value, not(equalTo(clone)));
    }
```

**Comment by timtebeek** [comment]
Looking closely at the packages used in your examples above I think this might be a duplicate of 
- https://github.com/openrewrite/rewrite-testing-frameworks/issues/519

Would you mind trying again with the [the snapshot snapshot](https://docs.openrewrite.org/reference/snapshot-instructions) once the fix for that issue is merged?

**Comment by Saturas89** [comment]
> Looking closely at the packages used in your examples above I think this might be a duplicate of
> 
> * [HamcrestMatcherToAssertJ does not support matchers from "org.hamcrest.core.*" #519](https://github.com/openrewrite/rewrite-testing-frameworks/issues/519)
> 
> Would you mind trying again with the [the snapshot snapshot](https://docs.openrewrite.org/reference/snapshot-instructions) once the fix for that issue is merged?

Yes, its already done. Thank you.
